python/lib/dcoscli/dcoscli/marathon/main.py

```diff
@@ -1233,7 +1233,7 @@ def debug_details(self, app_id, json_):
                 emitter, queued_app,
                 tables.queued_app_details_table, json_)
         else:
-            raise DCOSException("No apps found in Marathon queue")
+            raise DCOSException("Couldn't find app {} in Marathon queue".format(app_id))
```

Comment:
There is a linting error as the line is too long now:

> dcoscli/tables.py:835:80: E501 line too long (83 > 79 characters)

Reply:
Yep, just pushed a commit to make `flake8` happy